test/replication-luatest/bootstrap_strategy_test.lua

```diff
@@ -116,7 +116,7 @@ g_auto.test_join_checks_fullmesh = function(cg)
     end)
     t.assert(not server_is_ready(cg.server3), 'Server3 is dead')
     -- Server is dead, stopping it will fail.
```

Comment:
Please remove this comment. 

Also check:

- test/replication-luatest/gh_5295_split_brain_test.lua
- test/replication-luatest/gh_7253_election_long_wal_write_test.lua

Looks like you forgot to fix them.